Signal/src/ViewControllers/ConversationView/Cells/ConversationViewCell.m

```diff
@@ -30,6 +30,15 @@ - (CGSize)cellSizeForViewWidth:(int)viewWidth contentWidth:(int)contentWidth
     return CGSizeZero;
 }
 
+- (void)setIsCellVisible:(BOOL)isCellVisible
+{
+    _isCellVisible = isCellVisible;
+
+    if (isCellVisible) {
+        [self layoutIfNeeded];
```

Comment:
I'm not sure if this is a complete solution since we don't have repros for all cases, but it helps in the case I can repro.  I'm surprised it's necessary.

Reply:
LGTM